src/main/java/user_use_cases/UserDatabaseGateway.java

```diff
@@ -0,0 +1,76 @@
+package user_use_cases;
+
+import entities.User;
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.Scanner;
+
+public class UserDatabaseGateway {
```

Comment:
Could you add a user gateway interface named UserGatewayInterface, in order to keep CA? just simply add an interface then implement it in UserDataBaseGateway.